sdcm/cluster.py

```diff
@@ -2544,7 +2541,7 @@ def _update_db_packages(self, new_scylla_bin, node_list):
         self.log.debug('User requested to update DB packages...')  # pylint: disable=no-member
 
         seed_nodes = self.get_seed_nodes()
-        non_seed_nodes = [n for n in self.nodes if not n in seed_nodes]  # pylint: disable=no-member
+        non_seed_nodes = self.get_non_seed_nodes()
```

Comment:
when you will convert to property no need to have this local variable

Reply:
done